addOns/domxss/src/main/java/org/zaproxy/zap/extension/domxss/DomXssScanRule.java

```diff
@@ -476,10 +477,14 @@ private void scanHelper(String attackVector, String url) throws DomAlertExceptio
                 attributeId = element.getAttribute("id");
                 attributeName = element.getAttribute("name");
 
-                element.sendKeys(attackVector);
+                if (tagName.equals("input")) {
+                    element.sendKeys(attackVector);
+                }
+
                 element.click();
+
             } catch (UnhandledAlertException uae) {
-                Stats.incCounter("domxss.vulns.input2");
+                Stats.incCounter("domxss.vulns.possibleXSSTriggers2");
```

Comment:
I agree these names are more descriptive, but what if people are already using the existing ones?
No idea if thats the case, but if so then it would break things for them...

Reply:
Updated to possibleDomXSSTriggers2